nrf_802154/doc/feature_description.rst

```diff
@@ -242,7 +242,7 @@ Injecting the CSL Phase Information Element
 
 The driver can update the Coordinated Sampled Listening (CSL) phase in a transmitted frame at the moment of the frame transmission, by performing a CSL phase injection, for both data frames and enhanced ACK frames.
 
-The driver calculates the injected CSL phase value from the moment it ended the transmission of the last SHR symbol to the middle of the first pending delayed-reception window.
+The driver calculates the injected CSL phase value from the moment it ended the transmission of the PHR to the middle of the first pending delayed-reception window.
```

Comment:
Maybe it is clear, but for correctness sake, expand PHR to full name + "(PHR)" afterwards.

Reply:
Fixed